Files/Views/ModernShellPage.xaml.cs

```diff
@@ -1116,9 +1116,9 @@ public void Back_Click()
             Frame instanceContentFrame = ContentFrame;
             if (instanceContentFrame.CanGoBack)
             {
-                FilesystemViewModel.CancelLoadAndClearFiles();
                 var previousSourcePageType = instanceContentFrame.BackStack[instanceContentFrame.BackStack.Count - 1].SourcePageType;
                 SelectSidebarItemFromPath(previousSourcePageType);
+                InstanceViewModel.FolderSettings.IsLayoutModeChanging = previousSourcePageType != CurrentPageType;
                 instanceContentFrame.GoBack();
```

Comment:
What if instead of calling GoBack()/GoForward(), we got the NavigationArguments off the back stack and navigated to a desired layout mode? If I recall correctly, we should be able to remove entries off the back/forward stacks. It's up to you because I understand this will be difficult and potentially out of scope.

Reply:
@duke7573 that's possible too, but what benefits would it have?

Reply:
@gave92 It would fix issues like #801 which may be out of scope.

Reply:
A partial fix for #801 has been included